Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -19,6 +19,7 @@
   --color-primary-highlight: #218FC9;
   --color-primary-text: #FFFFFF;
   --color-highlight: #DEF3FE;
+  --color-sibling-hover: #f3faff;
```

Comment:
Considering the set of colors was created based on a color palette and design, I prefer not to add new colors, in particular colors used only once. Am pushing a version where the entire background is highlighted when hovering the label and only the respective OPTION is highlighted when hovering the ELEMENT element.

Reply:
Comment completed. Please review.

Reply:
Looks good